native/android/src/main/java/com/nozbe/watermelondb/Database.kt

```diff
@@ -15,8 +15,21 @@ class Database(private val name: String, private val context: Context) {
                 if (name == ":memory:" || name.contains("mode=memory")) {
                     context.cacheDir.delete()
                     File(context.cacheDir, name).path
+                } else if (name.contains("/") || name.contains("file")) {
```

Comment:
i think we want it to start with `/` or start with `file:`, no?

Reply:
The file path is` file:///data/user/0/com.mattermost.rnbeta/files/xxx.db`.

Let me change the `contains` to `startsWith`